docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx

```diff
@@ -58,6 +58,26 @@ We replaced following legacy Authenticator texts:
 
 If you were using `I18n` to translate those keys, please update your translations accordingly to match the new strings.
 
+#### 3. Bumped minimum Angular version to 12
+
+If your application is on Angular 11 or before, please migrate to Angular 12+ as per [official guide](https://update.angular.io).
```

Comment:
nit: `If your application is on Angular 11 or older, ...`

Reply:
```suggestion
If your application is on Angular 11 or older, please migrate to Angular 12+ as per [official guide](https://update.angular.io).
```